PoCs/kamaji/Metal3/generate-credentials.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eux
```

Comment:
Its a poc, but `-x` with generate creds script isn't good practice, unless you specifically want it to be shown.